.github/workflows/e2e.yaml

```diff
@@ -131,7 +146,8 @@ jobs:
         uses: ./.github/actions/e2e/slack/notify
         if: (success() || failure()) && github.event_name != 'workflow_run' && inputs.workflow_trigger != 'versionCompatibility'
         with:
-          url: ${{ secrets.SLACK_WEBHOOK_URL }}
+          cluster_name: ${{ steps.generate-cluster-name.outputs.CLUSTER_NAME }}
+          url: ${{ inputs.workflow_trigger == 'soak' || secrets.SLACK_WEBHOOK_SOAK_URL && secrets.SLACK_WEBHOOK_URL }}
```

Comment:
I'm not sure I understand the need of having another slack url just for soak. Are we creating two separate notification channels? Can you explain to me what this line does and add it into a comment?

Reply:
The plan here is that we can monitor the soak testing on a separate notification channel on the outset until we have proper monitoring and clean-up in place. I don't think we want to send this to the other notification channel since this is going to cause us to get notifications every 3h and is going to spam quite a bit, taking away from the other notifications that are due to periodics and repo merges.

Reply:
^^